src/v0/destinations/facebook_pixel/transform.js

```diff
@@ -313,17 +283,9 @@ const responseBuilderSimple = (
       category.standard = true;
     }
     if (Object.keys(customData).length === 0 && category.standard) {
-      throw new ErrorBuilder()
-        .setMessage(
-          `After excluding ${FB_PIXEL_DEFAULT_EXCLUSION}, no fields are present in 'properties' for a standard event`
-        )
-        .setStatus(400)
-        .setStatTags({
-          ...statTags,
-          meta:
-            TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.META.BAD_EVENT
-        })
-        .build();
+      throw new InstrumentationError(
+        `After excluding ${FB_PIXEL_DEFAULT_EXCLUSION}, no fields are present in 'properties' for a standard event`
+      );
```

Comment:
```suggestion
        `After excluding ${...FB_PIXEL_DEFAULT_EXCLUSION}, no fields are present in 'properties' for a standard event`
      );
```

Reply:
Why do we need this here? Also, this is not working and jest is also not able to run on adding this.

Reply:
ok I though without spreading it wont work